frontend/packages/integration-tests-cypress/tests/app/poll-console-updates.spec.ts

```diff
@@ -0,0 +1,99 @@
+import { checkErrors } from '../../support';
+import { warningToast } from '../../views/form';
```

Comment:
Not new, but I think `warningToast` will match any PF warning alert (inline, etc). We probably want something more specific so a different warning on the page doesn't match.

https://github.com/openshift/console/blob/1ec76caec2d8e287d03fff3c992805106aeab863/frontend/packages/integration-tests-cypress/views/form.ts#L7

Reply:
Good point.  Now checking for the refresh link instead.